src/org/zaproxy/zap/extension/ascanrules/TestRemoteFileInclude.java

```diff
@@ -247,7 +247,10 @@ public void scan(HttpMessage msg, String param, String value) {
             }
 
         } catch (Exception e) {
-            log.error("Error scanning parameters for Path Traversal: " + e.getMessage() + " Caused by: " + e.getCause());
+        	if (log.isDebugEnabled()) {

```

Comment:
With the log level change, should the exception type being caught changed to a more specific type? Like IOException?